src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -61,10 +61,10 @@ const AuthenticatorButton: UISchemaElementComponent<{
   const onValidationHandler = useOnSubmitValidation();
   const focusRef = useAutoFocus<HTMLButtonElement>(focus);
   const describedByIds = [
-    `${iconName}-ctaLabel`,
+    ariaDescribedBy,
     description && `${iconName}-description`,
     usageDescription && `${iconName}-usageDescription`,
-    ariaDescribedBy,
+    `${iconName}-ctaLabel`,
```

Comment:
I am re-ordering these because it makes more sense when the screenreader announces the texts this way.

Reply:
Could you give an example of a before and after?

Reply:
Could you give an example of a before and after?

Reply:
Yessir! So let's take the example in this screenshot. Looking at the Phone Authenticator, the aria-describedby value is correctly set to 

```
select-authenticator-authenticate_Title_Verify_its_you_with_a_security_method_1 select-authenticator-authenticate_Description_Select_from_the_following_options_2 phone_number_4-description phone_number_4-ctaLabel
```

Which means a screenreader will first read out the button label, then the Title (Verify its you with a security method), then the subtitle (Select from the following options), then the button description (if any), then the call to action label (i.e. Select). 

<img width="1463" alt="image" src="https://user-images.githubusercontent.com/97472729/210459307-259a6228-a8fc-41ea-9909-02bfc4d8339a.png">


Before this change, what would happen is it would read out the label, then the call to action text (in this case it would be "Select"), then the button description then finally the Title, subtitle, etc.